assets/js/main.js

```diff
@@ -53,10 +57,67 @@ document.getElementById('showcases-slideshow')
     if (communityElements.container && communityElements.template) {
       // Alter count to show all cards
       communityElements.count = participants.length;
+      createParticipantesCards(participants, emojis, communityElements, indexes=pageParticipants);
       communityElements.container.innerHTML = "";
       createParticipantesCards(participants, emojis, communityElements);
     }
-
   });
+})();
+
+// pagination
+
+function pagePagination(participants, currentPage, perPage) {
+  const paginationEl = document.getElementById("pagination");
+  const totalPages = Math.ceil(participants.length / perPage);
+
+  const paginationNextLi = document.createElement('li')
+  const paginationNextButton = document.createElement('a');
+  
+  paginationNextButton.textContent = "Next";
+  paginationNextButton.href = `?page=${currentPage + 1}`;
+  paginationEl.appendChild(paginationNextLi);
+  paginationNextLi.append(paginationNextButton);
+
+  if (currentPage === totalPages) {
+    paginationNextLi.style.display = "none";
+  };
 
-})();
\ No newline at end of file
+  for (let i = 0; i < totalPages; i++) {
+    const paginationLi = document.createElement('li');
+    const paginationLink = document.createElement('a');
+    
+    paginationLink.textContent = i + 1;
+    paginationLink.href = `?page=${i+1}`;
+    paginationEl.appendChild(paginationLi);
+    paginationLi.append(paginationLink);
+
+    if (currentPage == i + 1) {
+      paginationLink.classList.add("pagination-active");
+    };
+  }
+
+  const paginationPrevLi = document.createElement('li')
+  const paginationPrevButton = document.createElement('a');
+  
+  paginationPrevButton.textContent = "Previous";
+  paginationPrevButton.href = `?page=${currentPage - 1}`;
+  paginationEl.appendChild(paginationPrevLi);
+  paginationPrevLi.append(paginationPrevButton);
+
+  if (currentPage === 1) {
+    paginationPrevLi.style.display = "none";
+  };
+  
+  let firstParticipantIndex = currentPage * perPage -6;
+  let lastParticipantIndex = currentPage * perPage -1;
+  let pageParticipants = [];
+  
+  if (lastParticipantIndex >= participants.length) {
+    lastParticipantIndex = participants.length -1;
+  };
+  
+  for (let i = firstParticipantIndex; i <= lastParticipantIndex; i++) {
```

Comment:
Minor point, but this is doing the same job as the utility function genIndexes, so could be simplified to:
`return genIndexes(lastParticipantIndex+1, firstParticipantIndex);`